### ISSUE TITLE
The extension '3DTILES_content_gltf' was found, but not declared in 'extensionsUsed'

### ISSUE THREAD

**Issue by aleksmelnyk** [Geodan/pg2b3dm#117]
I have validated cesium tileset, and following errors:
```
{
  "date": "2023-09-30T05:30:59.368Z",
  "numErrors": 1,
  "numWarnings": 0,
  "numInfos": 0,
  "issues": [
    {
      "type": "EXTENSION_FOUND_BUT_NOT_USED",
      "path": "/extensionsUsed",
      "message": "The extension '3DTILES_content_gltf' was found, but not declared in 'extensionsUsed'",
      "severity": "ERROR"
    }
  ]
}
```
I have tried the tileset in cesium 1.108 and it was not even loaded. But it works fine in 1.101.

I far as I can see it is related to newest api changes in tileset.json?

thanks.

**Comment by bertt** [comment]
is this 3DTILES_content_gltf really required? What does it?

Anyway it should load without it in 1.108 see https://bertt.github.io/wageningen/

**Comment by bertt** [comment]
Looks like it's required see https://github.com/CesiumGS/3d-tiles/tree/main/extensions/3DTILES_content_gltf#optional-vs-required

Sample of combination with other extensions:

```
  "extensionsUsed": [ "3DTILES_content_gltf" ],
  "extensionsRequired" : [ "3DTILES_content_gltf" ],
  "extensions": {
        "3DTILES_content_gltf": {
            "extensionsUsed": ["EXT_mesh_gpu_instancing"]
        }
    },
```

**Comment by bertt** [comment]
Workaround: change asset.version from "1.0" to "1.1"  in tileset.json. 

Will be  fixed in next release.

**Comment by bertt** [comment]
fixed in release 1.8.3
